Frontend/src/components/TeacherPage.jsx

```
@@ -919,7 +1815,26 @@ export default function TeacherDashboard() {
+                  setProgressForm({
+                    attendance: '',
+                    academicPerformance: 'good',
+                    behavior: 'good',
+                    notes: ''
+                  });
+                }}
+                onClick={() => {
+                  setShowProgressForm(false);
+                  setSelectedStudent(null);
+                  setProgressForm({
+                    attendance: '',
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)